agent/agentssh/agentssh.go

```diff
@@ -449,7 +449,7 @@ func (s *Server) CreateCommand(ctx context.Context, script string, env []string)
 		}
 	}
 
-	cmd := exec.CommandContext(ctx, shell, args...)
+	cmd := pty.CommandContext(ctx, shell, args...)
```

Comment:
Suggestion: Perhaps we should discourage the use of non-context commands and rename this to `pty.Command`? Main reason stdlib has two is that it's a later addition.

Reply:
Yeah, but then it wouldn't be a drop-in replacement.

I like keeping the same API as the standard library so people will find it familiar, even though it's tempting to "improve" the API.